at_secondary/at_secondary_server/lib/src/refresh/at_refresh_job.dart

```diff
@@ -21,32 +22,37 @@ class AtRefreshJob {
   final logger = AtSignLogger('AtRefreshJob');
 
   /// Returns the list of cached keys
-  Future<List<dynamic>?> _getCachedKeys() async {
-    var keysList = await keyStore.getKeys(regex: CACHED);
+  Future<List<String>?> _getCachedKeys() async {
+    List<String>? keysList = keyStore?.getKeys(regex: CACHED);
```

Comment:
keystore can never be null on server.
getKeys() returns List<String>. List<String>? not needed

Reply:
@murali-shris I added this because `secondaryPersistenceStore.getSecondaryKeyStore()` returns a nullable `HiveKeystore`. So the return will be nullable. As the return was nullable, the `keyList` can be null. I can add a condition to check `keyStore` is null.